backend/src/application/controllers/rating/create-rating-controller.ts

```diff
@@ -26,19 +33,25 @@ export class CreateRatingController implements Controller {
 
     const { requesterId, value, committee, comment, ticketId } = req
 
+    const reviewer = await this.userRepository.loadById({ id: requesterId })
 
-    const reviewer: any = await AppDataSource.manager
-      .findOneByOrFail(User, { id: requesterId })
-
-    const ticket: any = await AppDataSource.manager
-      .findOneByOrFail(Ticket, { id: ticketId })
+    if (!reviewer) {
+      return badRequest(new UnauthorizedError)
+    }
+    
+    const ticket = await this.ticketRepository.loadTicketById({ id: ticketId })
+    
+    if (!ticket) {
+      return badRequest(new UnauthorizedError)
+    }
```

Comment:
Isso é firula mas tem uns espaços em branco aqui.

```suggestion
    if (!reviewer) {
      return badRequest(new UnauthorizedError)
    }

    const ticket = await this.ticketRepository.loadTicketById({ id: ticketId })

    if (!ticket) {
      return badRequest(new UnauthorizedError)
    }
```